### PR TITLE
changed layout to match "in patient"

### PR DESCRIPTION
**Pull request by kkyl**
- changed layout (most buttons have no function for now)
- search bar now also works if you press "Enter" aside from clicking "New" button
- New pending orders now display as a new box

FYI I removed the manually written CBC lab for now. Need to look into finding an API for getting all possible labs to order, otherwise we'll have to manually create our own json.

### CODE REVIEW

**Comment by avaidyam** [comment]
I made some of those changes this morning! Can you check the live site and compare? And I would suggest keeping the CBC for now since we actually need to show the lab order UI as part of the demo.

**Comment by kkyl** [comment]
I assume you still want the buttons from my version: 
image

also when you search an order, do labs typically appear as a separate table in the same dialog box or are search results mixed in as meds and labs together?

**Comment by avaidyam** [comment]
Absolutely correct! That's the button layout we want. I also updated the original issue with more screenshots - the non-medication orders are mixed in with other orders.

File: src/util/getRxTerms.js [hunk 1]
```diff
@@ -98,17 +117,45 @@ function formatRxTerms(data) {
...
+      return retVal;
+   })
+}
+
 export async function getRxTerms(searchTerm) {
   /** disabled nonfunctional try-catch to make eslint happy */
-  // try {
-  const rxtermsApiUrl = `https://clinicaltables.nlm.nih.gov/api/rxterms/v3/search?terms=${searchTerm}&ef=STRENGTHS_AND_FORMS,RXCUIS`;
+  // try 
+  const rxtermsApiUrl = `https://clinicaltables.nlm.nih.gov/api/rxterms/v3/search?terms=${searchTerm}&ef=STRENGTHS_AND_FORMS,RXCUIS&maxList=500`;
```
Comment (avaidyam) [inline src/util/getRxTerms.js:147]: Smart! Good idea to add this.

**Pull request merged**